device/isis/routerauth.yaml

```diff
@@ -0,0 +1,48 @@
+components:
+  schemas:
+    Router.IsisRouter.Authentication:
+      description: |-
+        This contains Isis router advanced properties.
+      type: object
+      properties:
+        ignore_receive_md5:
+          description: |-
+            The sequence number received for an MD5 key
+          type: boolean
+          default: true
+        area_auth_type:
+          description: |-
```

Comment:
is this required since the reference also has a description property (which seems to be more clear?)